src/ASM/LR/ASMu2Dmx.C

```diff
@@ -633,14 +646,22 @@ bool ASMu2Dmx::refine (const LR::RefineData& prm,
     for (size_t j = 0; j < m_basis.size(); ++j)
       if (j == bas)
         continue;
-      else if (line->span_u_line_)
-        m_basis[j]->insert_const_u_edge(line->const_par_,
-                                        line->start_, line->stop_,
-                                        line->multiplicity_);
-      else
-        m_basis[j]->insert_const_v_edge(line->const_par_,
-                                        line->start_, line->stop_,
-                                        line->multiplicity_);
+      else {
+        int mult = 1;
+        if (ASMmxBase::Type == ASMmxBase::REDUCED_CONT_RAISE_BASIS1 ||
+            ASMmxBase::Type == ASMmxBase::REDUCED_CONT_RAISE_BASIS2) {
+          int p = m_basis[bas]->order(line->span_u_line_ ? 0 : 1)-1;
```

Comment:
Actually, if the line is spanning the u-direction, then the reduced continuity is *across* it, i.e. in the v-direction. You will have to switch the polynomial degree you are looking up here.